src/website/app/pages/Tokens/index.js

```diff
@@ -0,0 +1,89 @@
+/* @flow */
+import React, { Component, Fragment } from 'react';
+import categorizedJsTokens from '../../categorizedJsTokens';
+import categorizedSassTokens from '../../categorizedSassTokens';
+import { createStyledComponent, pxToEm } from '../../../../library/styles';
+import Box from '../../../../library/Box';
+import { FormField } from '../../../../library/Form';
+import { RadioGroup } from '../../../../library/Radio';
+import _Callout from '../../Callout';
+import Button from '../../SiteButton';
+import Markdown from '../../Markdown';
+import VariableTable from '../../VariableTable';
+import VariableTableTitle from '../../VariableTableTitle';
+import content from './tokens.md';
+
+type Props = {};
+
+type State = {
+  format: Object,
+  value: string
+};
+
+const Callout = createStyledComponent(_Callout, ({ theme }) => ({
+  marginTop: theme.space_stack_xxl
+}));
+
+const REGEX_IS_COLOR = /^#/;
+
+const getColor = (theme, variable) =>
+  REGEX_IS_COLOR.test(theme[variable]) && theme[variable];
+
+const getValue = (theme, variable) => theme[variable];
+
+const formats = {
+  categorizedJsTokens,
+  categorizedSassTokens
+};
+
+const data = [
+  { label: 'JavaScript', value: 'categorizedJsTokens' },
+  { label: 'Sass', value: 'categorizedSassTokens' }
+];
+
+export default class Tokens extends Component<Props, State> {
+  state = { format: categorizedJsTokens, value: data[0].value };
+
+  render() {
+    return (
+      <div {...this.props}>
+        <Markdown scope={{ Button }}>{content}</Markdown>
+        <Callout title="Select a token format to display correct token names in the table below:">
+          <Box paddingTop={pxToEm(15)}>
```

Comment:
Why 15? We prefer to use theme variables (closest is `md`) for spacing, where possible.

Reply:
Sounds good. I was experimenting with how it looked compared to the table headings since it sits right above the first one. Good idea though, will update.